### PR TITLE
Saimon/create array with all drives

### PR DESCRIPTION
**Pull request by saimonation**

### CODE REVIEW

File: cterasdk/core/settings.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    def get_timezone(self):
+        """
+        Get timezone
+        """
+        return self._portal.get('/settings/timezone')
+
+    def set_timezone(self, timezone):
+        """
+        Get timezone
```
[the remark this answers is not in the document]
Comment (saimonation) [reply cterasdk/core/settings.py:28]: Done

File: cterasdk/core/ssl.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        """
+        Export the Portal SSL Certificate to a ZIP archive
+
+        :param str,optional destination:
+         File destination, defaults to the default directory
+        """
+        directory, filename = self._filesystem.split_file_directory_with_defaults(destination, 'certificate.zip')
+        logging.getLogger().info('Exporting SSL certificate.')
+        handle = self._portal.openfile('/admin/preview/exportCertificate', use_file_url=True)
+        filepath = FileSystem.instance().save(directory, filename, handle)
```
[the remark this answers is not in the document]
Comment (saimonation) [reply cterasdk/core/ssl.py:45]: Done

File: cterasdk/core/templates.py [hunk 3]
```diff
@@ -168,6 +189,51 @@ def remove_default(self, name, wait=False):
...
+        not_found = [template for template in templates if template not in portal_templates.keys()]
+        if not_found:
+            logging.getLogger().error('Could not find one or more templates. %s', {'templates': not_found})
+            raise CTERAException('Could not find one or more templates', None, templates=not_found)
+
+        policy = self.get_policy()
+
+        if apply_default is False:
+            policy.defaultTemplate = None
+        elif apply_default is True and default:
```
[the remark this answers is not in the document]
Comment (saimonation) [reply cterasdk/core/templates.py:222]: This configuration corresponds to a checkbox on a Portal on whether the default should be applied or not. So the Portal can keep the previously configured template and just turn off the default assignment.

File: cterasdk/core/settings.py [hunk 4]
```diff
@@ -25,7 +25,7 @@ def get_timezone(self):
 
     def set_timezone(self, timezone):
         """
-        Get timezone
+        Set timezone
```
Comment (ygalblum) [inline cterasdk/core/settings.py:28]: Document the timezone parameter

File: cterasdk/core/templates.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ygalblum) [reply cterasdk/core/templates.py:221]: OK

**Pull request merged**